core/api_threads.go

```diff
@@ -51,7 +51,9 @@ func (a *api) addThreads(g *gin.Context) {
 	}
 
 	if opts["schema"] != "" {
-		config.Schema.Id = opts["schema"]
+		config.Schema = &pb.AddThreadConfig_Schema{
```

Comment:
Was giving me issues because I think if `Schema` hasn't been initialized yet, trying to assign to `Id` panics. Pls check to make sure this is right (seems to work)?